cmd/lakectl/cmd/local.go

```diff
@@ -71,9 +72,19 @@ type syncFlags struct {
 	presign     bool
 }
 
-func getLocalSyncFlags(cmd *cobra.Command) syncFlags {
-	parallelism := Must(cmd.Flags().GetInt(localParallelismFlagName))
+func getLocalSyncFlags(cmd *cobra.Command, client *api.ClientWithResponses) syncFlags {
 	presign := Must(cmd.Flags().GetBool(localPresignFlagName))
+	presignFlag := cmd.Flags().Lookup(localPresignFlagName)
+	if !presignFlag.Changed {
```

Comment:
IIUC this means that if the user didn't set the flag we let the server decide if we use pre-sign or not, right? 
If that's the case I think it's a bit weird behavior because it's not aligned with other pre-sign commands with have in lakectl (i.e `fs cat` or `fs upload`). 
IMO we better omit this behavior. Thoughts?

Reply:
The difference here is that the pre-signed url flag is set to true by design. In the other commands you need to pass this flag explicitly.

Reply:
ohhh 💯